scripts/evaluation.py

```diff
@@ -84,4 +117,18 @@ def class_performance_ranks(ranker, eval_path):
             class_performance_ranks(ranker, eval_path_test)
 
     else:
-        print("code for classification")
+
+        if config["eval_on_test"]:
+            class_performance_classification(prediction_path_dev, valid_loader, dataset_valid, eval_path_dev)
+            class_performance_classification(prediction_path_test, test_loader, dataset_test, eval_path_test)
+            if argp.confusion_matrix:
+                conf_path_dev = str(Path(config["model_path"]).joinpath(config["save_name"] + "_confusion_dev.csv"))
+                conf_path_test = str(Path(config["model_path"]).joinpath(config["save_name"] + "_confusion_test.csv"))
+                confusion_matrix_classification(prediction_path_dev, valid_loader, dataset_valid, conf_path_dev)
+                confusion_matrix_classification(prediction_path_test, test_loader, dataset_test, conf_path_test)
```

Comment:
with more than three arguments, named parameter are more convenient for people who have not written the code.